plugins/scaffolder-common/src/TaskSpec.ts

```diff
@@ -21,11 +21,22 @@ import { JsonValue, JsonObject } from '@backstage/types';
  * stored in the database.
  *
  * @public
+ * @deprecated use templateInfo on the spec instead
  */
 export type TemplateMetadata = {
   name: string;
 };
 
+/**
+ * Information about a template that is stored on the scaffolder Spec
```

Comment:
```suggestion
 * Information about a template that is stored on a task specification
 ```

or something like that, was unclear what the wording referred to